Services/Certificate/classes/Placeholder/Values/class.ilScormPlaceholderValues.php

```diff
@@ -1,63 +1,27 @@
-<?php
+<?php declare(strict_types=1);
 /* Copyright (c) 1998-2018 ILIAS open source, Extended GPL, see docs/LICENSE */
 
 /**
  * @author  Niels Theen <ntheen@databay.de>
  */
 class ilScormPlaceholderValues implements ilCertificatePlaceholderValues
 {
-    /**
-     * @var ilLanguage|null
-     */
-    private $language;
-
-    /**
-     * @var ilDefaultPlaceholderValues|null
-     */
-    private $defaultPlaceHolderValuesObject;
-
-    /**
-     * @var ilCertificateDateHelper|null
-     */
-    private $dateHelper;
-
-    /**
-     * @var ilCertificateObjectHelper|null
-     */
-    private $objectHelper;
-
-    /**
-     * @var ilCertificateUtilHelper|null
-     */
-    private $utilHelper;
+    private ?ilLanguage $language;
+    private ?ilDefaultPlaceholderValues $defaultPlaceHolderValuesObject;
+    private ?ilCertificateDateHelper $dateHelper;
+    private ?ilCertificateObjectHelper $objectHelper;
+    private ?ilCertificateUtilHelper $utilHelper;
+    private ?ilCertificateObjectLPHelper $objectLPHelper;
+    private ?ilCertificateLPStatusHelper $lpStatusHelper;
```

Comment:
These types are not optional. Only the passed constructor arguments are optional.